src/lib.rs

```diff
@@ -1,7 +1,5 @@
 //! This library implements Nova, a high-speed recursive SNARK.
 #![deny(
-  warnings,
```

Comment:
can't we return back these lint checks?

Reply:
for some reasons my rust compiler fails with this, will look for workaround